src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java

```diff
@@ -209,4 +212,35 @@ public void testDreamSensitivity() throws Exception {
         // 78
         Assert.assertEquals(snpRecord.getTruePositives() + snpRecord.getFalseNegatives(), 78);
     }
+
+    @Test
+    public void testDoesNotCrashWithNO_VARIATIONAlleles() {
+        final String testDir = toolsTestDir + "concordance/";
```

Comment:
This appears elsewhere in this test class, so perhaps extract a static constant for it.

Reply:
Sounds good!  Fixed!